test/e2e/flowaggregator_test.go

```diff
@@ -456,26 +461,92 @@ func testHelper(t *testing.T, data *TestData, podAIPs, podBIPs, podCIPs, podDIPs
 		}
 	})
 
+	// Creating test client and server Pods for ToExternal tests
```

Comment:
mention in the comment that both clients are deployed on Node 0, which is the control-plane Node